memcached-operator/tools.go

```diff
@@ -0,0 +1,5 @@
+// +build tools
+
+// Place any runtime dependencies as imports in this file.
+// Go modules will be forced to download and install them.
+package tools
```

Comment:
Remove this file since `memcached-operator` doesn't have any runtime dependencies?

Reply:
The scaffold will add it by default. It was not added before because it is something which was introduced in the 0.9 version. So, should we not keep it?

Reply:
This is a nit, so not a merge blocker, but IMO the samples should reflect how we would actually develop the operator. If it were me developing the operator, I would probably remove this file from the project since there are not runtime dependencies. But that's my personal preference. No strong opinion on this.

Reply:
Removed 👍